app/_locales/en/messages.json

```diff
@@ -4249,9 +4249,6 @@
   "receive": {
     "message": "Receive"
   },
```

Comment:
This code is present on develop, and so it probably should not be deleted: https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L4323-L4325

Reply:
I agree with you that the develop shows as posted by you.

But in v12.4.2 has this changes - https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L4323-L4325

Hence I took the v12.4.2. Now my current question is how did this code get to hotfix only?

Reply:
ok thanks @danjm, I have address the conflict with the instruction you have provided.